lib/oli_web/components/delivery/students/students.ex

```diff
@@ -723,33 +742,171 @@ defmodule OliWeb.Components.Delivery.Students do
 
   def add_enrollments(%{add_enrollments_step: :step_2} = assigns) do
     ~H"""
+    <div class="px-4 flex flex-col space-y-4">
+      <div
+        :if={@add_enrollments_grouped_by_status[:non_existing_users] not in [[], nil]}
+        id="non_existing_users"
+      >
+        <p>
+          The following emails don't exist in the database. If you still want to proceed, an email will be sent and they
+          will become enrolled once they sign up. Please, review them and click on "Next" to continue.
+        </p>
+        <div>
+          <li class="list-none mt-4 max-h-80 overflow-y-scroll">
+            <%= for email <- @add_enrollments_grouped_by_status[:non_existing_users] do %>
+              <ul class="odd:bg-gray-200 dark:odd:bg-neutral-600 even:bg-gray-100 dark:even:bg-neutral-500 p-2 first:rounded-t last:rounded-b">
+                <div class="flex items-center justify-between">
+                  <p><%= email %></p>
+                  <button
+                    phx-click={
+                      JS.push("add_enrollments_remove_from_list",
```

Comment:
Just a thought, but I believe that we could avoid using the `JS` library when no JS is being used after the event is triggered. As a dev I'd expect to see some javascript code running but it is just Elixir/LiveView.